apps/web/src/components/devices/devices-overview-card.tsx

```diff
@@ -1,15 +1,27 @@
 import { DeviceContextProvider } from "@/hooks/device-hook";
+import { getActionSession } from "@/lib/auth/auth.action";
+import { getUserData } from "@/query/user";
 import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@energyleaf/ui/card";
 import { ErrorBoundary } from "@energyleaf/ui/error";
 import { Skeleton } from "@energyleaf/ui/skeleton";
 import { Suspense } from "react";
 import DeviceAddButton from "./device-add-button";
 import { DeviceDeleteDialog } from "./device-delete-dialog";
 import DeviceEditDialog from "./device-edit-dialog";
+import DevicesBadPowerEstimationHint from "./devices-bad-power-estimation-hint";
 import DevicesTable from "./devices-table";
 import DevicesTableError from "./table/devices-table-error";
 
-export default function DevicesOverviewCard() {
+export default async function DevicesOverviewCard() {
+    const { user } = await getActionSession();
```

Comment:
Bitte in Components getSession() benutzen.

Components/Frontend - getSession
Actions und API - getActionSession